templates/web/isleofwight/admin/triage/_inspect.html

```diff
@@ -1,16 +1,27 @@
 [% BLOCK category_list %]
 <select class="form-control" name="[% field_name %]" id="[% field_name %]">
-  [% IF NOT problem.category OR NOT categories_hash.${problem.category} %]
-    <optgroup label="[% loc('Existing category') %]">
-        <option selected value="[% problem.category | html %]">[% (problem.category_display OR '-') | html %]</option>
-    </optgroup>
-  [% END %]
   [% IF category_options.size %]
-    <optgroup label="[% loc('Available categories') %]">
-      [% FOREACH cat IN category_options %]
-        <option value="[% cat.category | html %]"[% ' selected' IF problem.category == cat.category %]>[% cat.category_display | html %]</option>
+        [%~ IF category_groups.size ~%]
+            [%~ FOREACH group IN category_groups ~%]
+                [% IF group.name %]<optgroup label="[% group.name %]">[% END %]
+                [% group_select = 0 %]
+                [% selected = 0 %]
+                [%~ FOREACH cat_op IN group.categories ~%]
+                    [% IF group_select == 0 AND problem.category == group.name %]
```

Comment:
Instead of the loop vars, could the IF on the option line not be `IF problem.category == cat_op.category OR problem.category == group.name`?

Reply:
Won't that select multiple items?

Reply:
I think the logic is the same as the existing code? Currently it says if the problem category matches cat_op.category or selected, which can be set if it matches the group name, so it'll also select both if there are one of each with the same name.

Reply:
If you have a Group called "Roads" and the problem is set to the category "Roads" then it will set `selected` for every category in that group, whereas I only want it to set `selected` for the first.

The idea is that the Confirm categories have groups that are named the same as the Triage categories

Reply:
Aha, I see what you mean, sorry for the misunderstanding.